assets/app/view/game/spreadsheet.rb

```diff
@@ -181,6 +207,9 @@ def render_titles
         end
         @extra_size = extra.size
 
+        extra_connection_title = @game.respond_to?(:connection_run) ? [h(:th, th_props[1], 'Conn.')] : []
+        extra_connection_sub = @game.respond_to?(:connection_run) ? [h(:th, '')] : []
```

Comment:
Why the extra-prefix if it’s called "connection_run" everywhere else?

Reply:
It's meant to convey that this is an optional column, but I could not use the "extras" variable as I wanted to place it just before the OR history. So I settled on using the extra-prefix to continue that style.

Reply:
Sure, if you think this is necessary => extra_connection_run… (it’s a run, not a connection!)